src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -245,13 +245,18 @@ export async function getReactNativeProjectAppVersion(versionSearchParams: Versi
 export function runReactNativeBundleCommand(bundleName: string, development: boolean, entryFile: string, outputFolder: string, platform: string, sourcemapOutput: string, extraBundlerOptions: string[]): Promise<void> {
   const reactNativeBundleArgs: string[] = [];
   const envNodeArgs: string = process.env.CODE_PUSH_NODE_ARGS;
+  let cliPath: string;
 
   if (typeof envNodeArgs !== "undefined") {
     Array.prototype.push.apply(reactNativeBundleArgs, envNodeArgs.trim().split(/\s+/));
   }
-
+  if (process.platform === "darwin") {
```

Comment:
Usually linux & macOS are quite similar in this kind of things while Windows can differ, has this been tested on all 3?

Reply:
This has been tested on macOS and windows, but I think that in this case on linux it will work just as well as on macOS.

Reply:
Right, so then (Linux and macOS behaving the same) it makes more sense to have the special case handle Windows, and have the rest behave the same.

Reply:
Yea I think you're right, I updated it.